index.js

```diff
@@ -7,12 +7,37 @@ function ask(questionText) {
   });
 }
 
+let loNum = 0;
+let hiNum = 100;
+let yesOrNo = "n";
+let numTries = 0;
+
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  console.log(
+    "\nPlease think of a number between 1 and 100 (inclusive)\nand I will try to guess it.\n"
+  );
+  let secretNumber = await ask(
+    "What is your secret number? I won't peek, I promise... "
+  );
+  console.log("\nOK, let's begin the game.\n");
+
+  while (yesOrNo === "n") {
+    guess = Math.floor((loNum + hiNum) / 2);
+    numTries = numTries + 1;
+    let yesOrNo = await ask("Is your number " + guess + "? (y/n) ");
+    if (yesOrNo === "y") {
```

Comment:
consider sanitizing your input here to handle edge cases, such as someone typing in a capital "Y" or the word "yes"